SparkPost/2web2ui
# FE-1170 - [Dashboard UX Redesign] Incorporate conditional rendering per user role

[FE-1170](https://sparkpost.atlassian.net/browse/FE-1170)

**Note:** Still waiting to merge this one in - this branch is based on https://github.com/SparkPost/2web2ui/pulls

### What Changed
- Adds conditional rendering based on roles and grants to the Dashboard container component.
- Adds supporting tests for conditional rendering states
- Moves some of the dashboard content around to match the mocks

### How To Test

- Verify `/dashboardV2` with each user role, and verify that these sections render according to the mocks attached in the ticket

### To Do

- [x] Wait for https://github.com/SparkPost/2web2ui/pulls to be merged
